### DISCUSSION TITLE
unRAID users - do you record to a cache drive?

### DISCUSSION THREAD

**Discussion by wayner9** [OpenSageTV/Community#98] [General]
I have been wondering if it make sense to have my recordings go to the cache drive as my cache drive is an NVME SSD whereas my pool drives are physical drives, and the parity operation will slow writes to the pool.  I would also use the cache for all media files, like when I rip DVDs or acquire other video files, so that they would initially go on the cache drive which should allow for faster copies.

What do others do?  Is there a benefit to storing recordings to the cache, and then have mover move them to the pool overnight?

**Comment by something-fishy** [comment]
I have appdata on the cache drive proper (SSD) and recordings on a pool drive (HDD). I don't involve SageTV with the array which consequently stays spun down most of the time (its content is mainly static and backups). 

Obviously this leaves SageTV recordings vulnerable to the pool drive failing but that's a risk I'm willing to take for the power savings and given that SageTV is recording free to air TV channels which now all offer catchup app-based archives.

**Reply by wayner9** [reply]
What do you mean by a pool drive in this context?  Do you have a separate "recording" pool that is separate from your array and Cache pool?

Why worry so much about power consumption?  Don't hard drives use about 7W.  So assuming that they are using that much power 24x7 that is about 5kWh per month (7Wx24hours/day*30days/month/1000W/kW).  That's not much to be worried about, is it?

**Reply by something-fishy** [reply]
If you add a drive to the "cache pool" but don't create mover rules to it, it functions as a standalone drive outside the array. In my case the "recordings' user share writes only the "Sagedata" drive and are not moved anywhere else.

In my case I'm more worried about heat than power use per se. The server (and an HDBaseT matrix and my network switch) are all in a small closet without real cooling. So keeping the drives spun down reduces the heat generation and/or fan noise.

**Reply by something-fishy** [reply]
@KeithAbbott 
Nothing I think. However when I moved SageTV from the dedicated Win7 PC I used to an Unraid docker cache pools were a thing (I've used Unraid for many years with a simple single cache drive, but only adopted docker containers recently.

**Comment by wayner9** [comment]
`If you add a drive to the "cache pool" but don't create mover rules to it, it functions as a standalone drive outside the array. In my case the "recordings' user share writes only the "Sagedata" drive and are not moved anywhere else.`

It might just be semantics, but I think that such drives aren't really in the cache pool.  Starting with unRAID 6.9 you could create multiple pools, so you can have a cache pool, and then other pools for other purposes.
